core/src/world/npc/view.rs

```diff
@@ -97,7 +97,13 @@ impl<'a> fmt::Display for DetailsView<'a> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let Self { npc, relations } = self;
 
-        writeln!(f, "<div class=\"thing-box npc\">\n")?;
+        writeln!(
+            f,
+            "<div class=\"thing-box npc\"{}>\n",
+            npc.uuid
+                .clone()
```

Comment:
Try with `as_ref()` instead.

Reply:
I'll be sure to use `as_ref()` where possible. I've gotten perhaps too comfortable with throwing `clone()` everywhere. :)